### PR TITLE
Add PHP files to language packs

### PR DESCRIPTION
**Pull request by swissspidy**
Ticket: https://meta.trac.wordpress.org/ticket/7296

### CODE REVIEW

**Comment by dd32** [comment]
I haven't reviewed this, but it looks like it'll work.

GlotPress will need to make an alpha release before we can merge this though; as the `php` format isn't yet on https://plugins.trac.wordpress.org/browser/glotpress/trunk/gp-includes/formats

File: wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/inc/cli/class-export.php [hunk 1]
```diff
@@ -102,6 +102,17 @@ public function __invoke( $args, $assoc_args ) {
...
 
+			// Create PHP file.
+			$php_file = "{$wp_locale}.l10n.php";
+			if ($context) {
+				$php_file = "$context-{$php_file}";
+			}
+			$php_file = "{$dest}/{$php_file}";
+			$result = $this->build_php_file( $gp_project, $gp_locale, $translation_set, $po_entries, $php_file );
+			if ($result) {
+				$files[] = $php_file;
```
Comment (dd32) [inline wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/inc/cli/class-export.php]:
```suggestion
			// Create PHP file.
			$php_file = "{$wp_locale}.l10n.php";
			if ( $context ) {
				$php_file = "$context-{$php_file}";
			}
			$php_file = "{$dest}/{$php_file}";

			$result = $this->build_php_file( $gp_project, $gp_locale, $translation_set, $po_entries, $php_file );
			if ( $result ) {
				$files[] = $php_file;
```

Some nitpicky whitespace.

**Review by dd32 — APPROVED** [review]

**Comment by swissspidy** [comment]
> GlotPress will need to make an alpha release before we can merge this though; as the `php` format isn't yet on [plugins.trac.wordpress.org/browser/glotpress/trunk/gp-includes/formats](https://plugins.trac.wordpress.org/browser/glotpress/trunk/gp-includes/formats)

This is now complete & the new format is available ✅

**Review by amieiro — APPROVED** [review]
I have tested it and it works fine. 
As we have released a new GlotPress version ([4.0.0-beta.2](https://github.com/GlotPress/GlotPress/releases/tag/4.0.0-beta.2)) with the new PHP format, this PR is ready to be released.

**Comment by amieiro** [comment]
I have [committed](https://meta.trac.wordpress.org/changeset/13161) this PR to Meta, so I close it.

**Pull request closed without merging**